service/db/src/main/java/gov/va/vro/service/db/SaveToDbServiceImpl.java

```diff
@@ -30,6 +40,74 @@ public Claim insertClaim(Claim claim) {
     return claim;
   }
 
+  @Override
+  public AssessmentResult insertAssessmentResult(
+      UUID id,
+      String assessmentResult,
+      String veteranIcn,
+      String claimSubmissionId,
+      String idType,
+      String diagnosticCode)
+      throws NoSuchFieldException {
+    ClaimEntity claimEntity =
+        claimRepository
+            .findById(id)
+            .orElseThrow(
+                () ->
+                    new NoSuchFieldException(
+                        "Could not match claim ID {" + id.toString() + "} in DB."));
+    Map<String, Object> evidence = new HashMap<>();
+    try {
+      evidence = objMapper.readValue(assessmentResult, Map.class);
+      if (evidence.isEmpty()) {
+        throw new NoSuchFieldException("Could not map assesmentResult, evidence summary is null.");
+      }
+    } catch (Exception e) {
+      log.error("Could not map assesmentResult, evidence summary is null.");
+    }
+    Map<String, Object> defaultEvidence = new HashMap<>();
+    defaultEvidence.put("medicationsCount", 0);
```

Comment:
why do we have a default here. there might be possibly conditions that do not use medications.

Reply:
I had set it to null but @dimitri-amida suggested this was a bad idea.  Should I make it ("default", 0)?